package Exceptions;

public class CheckedExceptions {
	
	public static void main(String[] args) throws Exception{
		
		
		System.out.println("Programming has started");
		System.out.println("Program is in progress");
		
//		try {
			Thread.sleep(5000);
//		} catch (InterruptedException e)
//		{
//			System.out.println("Inside catch block");
//			System.out.println(e.getMessage());
//		}
		
		
		System.out.println("Program has completed");
		System.out.println("Program is exited");
	}

}
